easy-ui-react/src/Modal/Modal.module.scss

```diff
@@ -0,0 +1,139 @@
+@use "../styles/common" as *;
+@use "../styles/unstyled";
+
+.underlayBg {
+  @include component-token("modal", "underlay-edge-max-height", 72px);
+  @include component-token("modal", "underlay-edge-min-height", 2.5vh);
```

Comment:
**question**: how was the min height here calculated?

Reply:
it's computed from the requirement in Figma for the max height of the modal to be "95% of the screen height" which leaves each top/bottom edge 2.5% or 2.5vh. let me know if that makes sense.

<img width="718" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/52d4c493-06e6-419e-95f5-8e3d71a16804">

Reply:
makes sense!